spec/facility_spec.rb

```diff
@@ -23,4 +23,276 @@
+      expect(facility_1.administer_road_test(registrant_1)).to eq(false)
+      expect(facility_1.add_service('Road Test')).to eq(["Written Test", "Road Test"])
+      expect(facility_1.administer_road_test(registrant_1)).to eq(true)
+      expect(registrant_1.license_data).to eq({:written=>true, :license=>true, :renewed=>false})
+      expect(facility_1.administer_road_test(registrant_2)).to eq(true)
+      expect(registrant_2.license_data).to eq({:written=>true, :license=>true, :renewed=>false})
+    end
+
+    it 'can renew license' do
+      registrant_1 = Registrant.new('Bruce', 18, true )
+      registrant_2 = Registrant.new('Penny', 16 )
+      registrant_3 = Registrant.new('Tucker', 15 )
+      facility_1 = Facility.new({name: 'DMV Tremont Branch', address: '2855 Tremont Place Suite 118 Denver CO 80205', phone: '(720) 865-4600'})
+      facility_2 = Facility.new({name: 'DMV Northeast Branch', address: '4685 Peoria Street Suite 101 Denver CO 80239', phone: '(720) 865-4600'})
+
+      registrant_1.license_data
+      registrant_1.permit?
+      facility_1.administer_written_test(registrant_1)
+      registrant_1.license_data
+      facility_1.add_service('Written Test')
+      facility_1.administer_written_test(registrant_1)
+      registrant_1.license_data
+      registrant_2.age
+      registrant_2.permit?
+      facility_1.administer_written_test(registrant_2)
+
+      registrant_2.earn_permit
+      facility_1.administer_written_test(registrant_2)
+      registrant_2.license_data
+      registrant_3.age
+      registrant_3.permit?
+      facility_1.administer_written_test(registrant_3)
+
+      registrant_3.earn_permit
+      facility_1.administer_written_test(registrant_3)
+      registrant_3.license_data
+
+      facility_1.administer_road_test(registrant_3)
+      registrant_3.earn_permit
+      facility_1.administer_road_test(registrant_3)
+      registrant_3.license_data
+      facility_1.administer_road_test(registrant_1)
+      facility_1.add_service('Road Test')
+      facility_1.administer_road_test(registrant_1)
+      registrant_1.license_data
+      facility_1.administer_road_test(registrant_2)
+      registrant_2.license_data
+
+      expect(facility_1.renew_drivers_license(registrant_1)).to eq(false)
+      expect(facility_1.add_service('Renew License')).to eq(["Written Test", "Road Test", "Renew License"])
+      expect(facility_1.renew_drivers_license(registrant_1)).to eq(true)
+      expect(registrant_1.license_data).to eq({:written=>true, :license=>true, :renewed=>true})
+      expect(facility_1.renew_drivers_license(registrant_3)).to eq(false)
+      expect(registrant_3.license_data).to eq({:written=>false, :license=>false, :renewed=>false})
+      expect(facility_1.renew_drivers_license(registrant_2)).to eq(true)
+      expect(registrant_2.license_data).to eq({:written=>true, :license=>true, :renewed=>true})
+    end
+  end
 end
+
+describe 'helper methods' do
```

Comment:
Great job **unit testing** your helper methods and using `describe` to organize this file.